sources/packages/load-test/src/_tests_/institution/institution-get-profile/institution-get-profile.test.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * Methods like setup and default are part of the K6 life cycle.
+ * @see https://k6.io/docs/using-k6/test-lifecycle
+ * The exported options are part of the K6 configuration.
+ * @see https://k6.io/docs/using-k6/k6-options
+ */
+import { check, sleep } from "k6";
+import { getAPICall } from "../../../utils/sims-api/sims-api";
+import { UserPasswordCredential } from "../../../utils/auth";
+import { getInstitutionAdminCredentials } from "../../../utils/sims-api";
+
+interface SetupData {
+  credentials: UserPasswordCredential;
+}
+
+/**
+ * Define test-run behavior.
+ * Ramping up to 120 during 10s to allow Keycloak to
+ * digest the requests. When all VUs are created at the
+ * same time it was resulting on an Keycloak internal
+ * server error (500).
+ */
+export const options = {
+  stages: [
+    { target: 120, duration: "10s" },
```

Comment:
for my understanding what is `target` here

Reply:
same question for me. is it 120 requests?

Reply:
During 10 seconds increase the number of VUs to 120.